packages/graph/hash_graph/bench/benches/representative_read/mod.rs

```diff
@@ -0,0 +1,115 @@
+mod knowledge;
+mod ontology;
+
+use std::{collections::HashMap, str::FromStr};
+
+use criterion::{BenchmarkId, Criterion};
+use criterion_macro::criterion;
+use graph::{
+    knowledge::{Entity, EntityId},
+    ontology::AccountId,
+    store::{AccountStore, EntityStore},
+};
+use graph_test_data::{data_type, entity, entity_type, link_type, property_type};
+use rand::{seq::IteratorRandom, thread_rng};
+use type_system::uri::{BaseUri, VersionedUri};
+use uuid::Uuid;
+
+use crate::util::{setup, StoreWrapper};
+
+/// DOC - TODO
+struct Samples {
+    pub entities: HashMap<AccountId, HashMap<VersionedUri, Vec<EntityId>>>,
+}
+
+/// Sets up the sample "representative" environment in which operations are benchmarked.
+///
+/// This initializes the database for all benchmarks within this module, and therefore should be a
+/// single point to swap out the seeding of test data when we can invest time in creating a
+/// representative environment.
+async fn seed_db(store_wrapper: &mut StoreWrapper) -> Samples {
+    // TODO: We'll want to test distribution across accounts
```

Comment:
I'll make and attach an asana task if people agree this is worth doing at some point (probably when we have sharding in place)